conans/model/conf.py

```diff
@@ -124,6 +124,7 @@
     "tools.build:compiler_executables": "Defines a Python dict-like with the compilers path to be used. Allowed keys {'c', 'cpp', 'cuda', 'objc', 'objcxx', 'rc', 'fortran', 'asm', 'hip', 'ispc'}",
     "tools.build:cxxflags": "List of extra CXX flags used by different toolchains like CMakeToolchain, AutotoolsToolchain and MesonToolchain",
     "tools.build:cflags": "List of extra C flags used by different toolchains like CMakeToolchain, AutotoolsToolchain and MesonToolchain",
+    "tools.build:asmflags": "List of extra assembly flags used by different toolchains like CMakeToolchain, AutotoolsToolchain and MesonToolchain",
```

Comment:
Needs to be fixed:
- Add "experimental"
- Clarify it only applies to CMakeToolchain, the other toolchains didn't implement it

Reply:
Would this be good for you?
`(experimental) List of extra assembly flags used by the CMakeToolchain generator`

Reply:
yes, thanks!